src/notation/view/notationviewinputcontroller.cpp

```diff
@@ -722,6 +722,10 @@ void NotationViewInputController::mouseMoveEvent(QMouseEvent* event)
         m_view->moveCanvas(dragDelta.x(), dragDelta.y());
         m_isCanvasDragged = true;
     }
+    if (m_view->isNoteEnterMode()) {
+        m_view->moveCanvas(dragDelta.x(), dragDelta.y());
+        m_isCanvasDragged = true;
+    }
```

Comment:
Note that this piece of code can be simplified as just 
```c++
    m_view->moveCanvas(dragDelta.x(), dragDelta.y());
    m_isCanvasDragged = true;
```
without any if statements (since `isNoteEnterMode == m_view->isNoteEnterMode()`)

Also, in note input mode I think this should only happen for middle mouse button. Wouldn't it affect left mouse button too, with the current code?